addons/account/i18n/account.pot

```diff
@@ -2348,6 +2348,10 @@ msgstr ""
 #: model:ir.model.fields,field_description:account.field_account_move_line__analytic_distribution_stored_char
 #: model:ir.model.fields,field_description:account.field_account_reconcile_model_line__analytic_distribution_stored_char
 msgid "Analytic Distribution Stored Char"
+
```

Comment:
Nearly there, I think you lost a `msgstr ""` here :smile:

Reply:
O boy :man_facepalming: I'm gonna need some coffee :laughing: